interp/tests/simple_tests/simple_neq.expect

```diff
@@ -1,25 +1,38 @@
-{
-  "main": {
-    "_1_1": {
-      "out": 1
-    },
-    "const0": {
-      "out": 10
-    },
-    "const1": {
-      "out": 10
-    },
-    "neq0": {
-      "left": 10,
-      "out": 0,
-      "right": 10
-    },
-    "reg0": {
-      "clk": 0,
-      "done": 1,
-      "in": 0,
-      "out": 0,
-      "write_en": 1
+[
+  [
```

Comment:
I would've expected this to be an object with "Ports:" and "Memory State:" being field names. I would also align the fields names to be similar to `fud --to dat` (so naming the field `memories` and the other field name that I don't remember).

Reply:
Yeah a dummy struct probably makes more sense, I'll change that in a bit

Reply:
Done: see most recent commit